### PR TITLE
create new page for registration 

### LINKED ISSUE #11: Create new page for registration
### Create registration page in the client side

Page must consist registration blank for new user.
Also link to login page
Google auth button

### PR DESCRIPTION
**Pull request by Vova-Romaniuk**

### CODE REVIEW

**Review by maryanchuk04 — APPROVED** [review]
Great job!!!

File: StudCity.Frontend/src/pages/registration/Registration.jsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+							type = "email" 
+							required = {true}
+						/>
+						<Label>Password</Label>
+						<PasswordTextField 
+							placeholder = "Enter your password"
+							className = "" 
+							required = {true}
+							setDisabled = {setDisabled}
+							handleChange = {(evnt) => setPasswords({ ...passwords,
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/pages/registration/Registration.jsx]: please create a function at the top of this component, and pass it as a prop
PS: in other places as well

File: StudCity.Frontend/src/UI/Fields/PasswordTextField.jsx [hunk 2]
```diff
@@ -1,76 +1,62 @@
 import React, { useState } from "react";
 
-function PasswordTextField({ className = "", placeholder, required, setDisabled }) {
+function PasswordTextField({ className = "", placeholder, required, setDisabled, handleChange, secondValue = "", validate }) {
 	const [hidden, setHidden] = useState(false);
 	const [passwordError, setPasswordErr] = useState("");
 	const primaryGray = "#506466";
+
+	const onChange = (evnt) => {
```
Comment (Serhiy-dotcom) [inline StudCity.Frontend/src/UI/Fields/PasswordTextField.jsx]: remove this function and use handleValidation instead

**Review by Serhiy-dotcom — CHANGES_REQUESTED** [review]
Please fix changes

File: StudCity.Frontend/src/UI/Fields/PasswordTextField.jsx [hunk 3]
```diff
@@ -1,76 +1,59 @@
 import React, { useState } from "react";
 
-function PasswordTextField({ className = "", placeholder, required, setDisabled }) {
+function PasswordTextField({ className = "", placeholder, required, setDisabled, handleChange, secondValue = "", validate }) {
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/UI/Fields/PasswordTextField.jsx:3]: You can add not required params in the end of props

File: StudCity.Frontend/src/pages/registration/Registration.jsx [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+import { passwordMatchValidation, passwordValidation } from "../../utils/validators/validators";
+
+function Registration() {
+	const [disabled, setDisabled] = useState(false)
+	const [passwords, setPasswords] = useState({
+		password: "",
+		confirmPassword: ""
+	})
+
+	const handleChange = (evnt) => {
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/pages/registration/Registration.jsx:16]: What is this??? 
You using this in two different component...

**Review by Serhiy-dotcom — APPROVED** [review]

**Review by maryanchuk04 — APPROVED** [review]
so bad, u can better.

**Pull request merged**